recipes/mysqld_exporter.rb

```diff
@@ -65,4 +65,13 @@
      service "Monitoring"
      restart_agent false
    end
-end
\ No newline at end of file
+end
+
+if exists_local('consul', 'master') or exists_local('consul', 'slave')
+  # Register MySQL exporter with Consul
+  consul_service "Registering MySQL exporter with Consul" do
+    service_definition "mysql-exporter-consul.hcl.erb"
+    restart_consul false
```

Comment:
This should be `reload_consul` instead https://github.com/logicalclocks/consul-chef/blob/master/resources/service.rb#L5

Reply:
Fixed